src/services/castleads.pas

```diff
@@ -313,35 +334,36 @@ procedure TAds.TAdNetworkHandler.ReinitializeJavaActivity(Sender: TObject);
     while waiting for ad to finish. Reproduce: turn on "kill app when sending to bg"
     in Android debug options, then use "watch ad" and return to game. }
   if FFullScreenAdVisible then
-    FullScreenAdClosed(false);
+    FullScreenAdClosed(wsUnknownError);
   Parent.FBannerSize := TRectangle.Empty;
```

Comment:
Maybe make it `wsApplicationReinitialized` or something like that. Although it's pretty obscure case, so I'm not sure about my own advise, it can stay `wsUnknownError`...

Reply:
I think we can add `wsApplicationReinitialized`. It can be helpful in debug.